pkg/security/securitycontextconstraints/byrestrictions.go

```diff
@@ -116,6 +114,9 @@ func volumePointValue(scc *securityapi.SecurityContextConstraints) points {
 		default:
 			hasNonTrivialVolume = true
 		}
+		if hasHostVolume {
```

Comment:
@php-coder check

Reply:
@mfojtik check passed but in this case we can even use `return`